src/FRP/Event/Class.purs

```diff
@@ -4,7 +4,7 @@ module FRP.Event.Class
   , folded
   , count
   , mapAccum
-  , mapMaybe
+  , module Data.Filterable
```

Comment:
Could you please move this to the bottom of the list?

Reply:
Of course!